validators/builtin-validators.mdx

```diff
@@ -287,6 +288,55 @@ Example:
 
 You can find this example in [Javascript](https://github.com/Lemoncode/fonk/tree/master/examples/docs/api/js/email) and [Typescript](https://github.com/Lemoncode/fonk/tree/master/examples/docs/api/ts/email).
 
+## Array
+
+This validator allows you to add support to validation to array form fields, it work as an inception,
```

Comment:
Add s in -> it **works** as an inception,

Reply:
done